cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt

```diff
@@ -533,8 +533,8 @@ script "js_azure_virtual_machines_with_metrics", type: "javascript" do
       "metricnames": "OS Disk IOPS Consumed Percentage,OS Disk Bandwidth Consumed Percentage,Data Disk IOPS Consumed Percentage,Data Disk Bandwidth Consumed Percentage",
       "timespan": timespan,
       "$filter": "LUN eq '*'",
-      "aggregation": "Average, Maximum",
-      "interval": "FULL",
+      "aggregation": "Average, Maximum, Minimum",
+      "interval": "PT1H",
```

Comment:
This should be a parameter, similar to what you see here: https://github.com/flexera-public/policy_templates/blob/d4c8705649ebbc65a29ac6027818d16a95e97e19/cost/azure/rightsize_compute_instances/azure_compute_rightsizing.pt#L174

The above policy should be used as a guide for how to implement this. We want all of the policies to function as similarly to each other as possible.

Reply:
Thank you @XOmniverse , I addressed this change.